stracker/stracker/settings.py

```diff
@@ -44,6 +44,13 @@
     'django.contrib.messages',
     'django.contrib.staticfiles',
     "django_extensions",
+    "core",
+    "items",
+    "locations",
+    "shelves",
+    "users",
+    "utils",
```

Comment:
core modülü var utils modülüne gerek var mı?

Reply:
Telegram grubunda en son atılan diagramda core/utils applere eklenebilir diye not gördüm. Onun için yaptım yani geçerli bir sebebi yok :)